README.md

```diff
@@ -49,8 +49,6 @@ The master branch works with **PyTorch 1.3+**.
 Supported backbones for human pose estimation:
 
 - [x] [AlexNet](configs/top_down/alexnet/README.md)
-- [x] [CPM](configs/top_down/cpm/README.md)
```

Comment:
why delete them

Reply:
They are also listed in `Supported methods`...

Reply:
make sense